addons/account/tests/common.py

```diff
@@ -1021,6 +1040,14 @@ def sort_lines(lines):
         self.assertRecordValues(move, [expected_move_values])
 
 
+@tagged('post_install', '-at_install')
+class AccountTestInvoicingHttpCommon(HttpCase, AccountTestSetup):
```

Comment:
the file already contains a lot of classes. Please docstring (and/or cleanup now or later)